verticals/vertical-ats/router.ts

```diff
@@ -0,0 +1,47 @@
+import type {AdapterFromRouter, RouterMeta} from '@openint/vdk'
+import {
+  proxyCallAdapter,
+  trpc,
+  verticalProcedure,
+  z,
+  zPaginatedResult,
+  zPaginationParams,
+} from '@openint/vdk'
+import adapters from './adapters'
+import * as unified from './unifiedModels'
+
+export {unified}
+
+function oapi(meta: NonNullable<RouterMeta['openapi']>): RouterMeta {
+  return {openapi: {...meta, path: `/verticals/ats${meta.path}`}}
+}
+
+const procedure = verticalProcedure(adapters)
+
+export const atsRouter = trpc.router({
+  listJobs: procedure
+    .meta(oapi({method: 'GET', path: '/jobs'}))
```

Comment:
By convention we use singular rather than plural in the path so we don't have to worry about weird "pluralization" rules in the English language. This goes for both api and database.